sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -246,11 +246,17 @@ statement
     | SET TIME ZONE interval                                           #setTimeZone
     | SET TIME ZONE timezone=(STRING | LOCAL)                          #setTimeZone
     | SET TIME ZONE .*?                                                #setTimeZone
+    | SET quotedConfigKey (EQ value=.*)?                               #setQuotedConfiguration
     | SET .*?                                                          #setConfiguration
+    | RESET quotedConfigKey                                            #resetQuotedConfiguration
     | RESET .*?                                                        #resetConfiguration
     | unsupportedHiveNativeCommands .*?                                #failNativeCommand
     ;
 
+quotedConfigKey
```

Comment:
hmm, is it necessary to create an alias? How about `SET key= quotedIdentifier (EQ value=.*)?`

Reply:
Yea, I did so first, but that suggested definition did not invoke `PostProcessor.exitQuotedIdentifier` to split backquotes;
```
-    | SET quotedConfigKey (EQ value=.*)?                               #setQuotedConfiguration
+    | SET key=quotedIdentifier (EQ value=.*)?                          #setQuotedConfiguration

   override def visitSetQuotedConfiguration(ctx: SetQuotedConfigurationContext)
     : LogicalPlan = withOrigin(ctx) {
-    val keyStr = ctx.quotedConfigKey().getText
+    val keyStr = ctx.key.getText

assertEqual("SET `spark.sql.    key`=value",
  SetCommand(Some("spark.sql.    key" -> Some("value"))))
fo] - Report Error for invalid usage of SET command *** FAILED *** (106 milliseconds)
[info]   == FAIL: Plans do not match ===
[info]   !SetCommand (`spark.sql.    key`,Some(value))   SetCommand (spark.sql.    key,Some(value)) (PlanTest.scala:157)
[info]   org.scalatest.exceptions.TestFailedException:
...
```
So, we need replace("`", "") for that approach. Please check the lates commit.